src/pages/NewSchedule.tsx

```diff
@@ -8,35 +11,75 @@ import { NickNameForm } from '../components/features/CreateScheduleForm/NickName
 import { ScheduleInputForm } from '../components/features/CreateScheduleForm/ScheduleInputForm';
 
 export const NewSchedule = () => {
-  const { Funnel, setStep } = useFunnel(newScheduleStepNames);
+  const accessToken = localStorage.getItem('accessToken');
+  const { uuid } = useParams();
 
-  const [newSchedule, setNewSchedule] = useState<newSchedule>({
+  const navigate = useNavigate();
+  const { Funnel, step, setStep } = useFunnel(newScheduleStepNames);
+
+  useEffect(() => {
+    if (accessToken) {
+      setStep('일정입력');
+    }
+  }, [accessToken, setStep]);
+
+  const [nickName, setNickName] = useState('');
+  const [schedule, setSchedule] = useState<newSchedule>({
     dateOfScheduleList: [],
-    nickName: '',
   });
 
-  const updateSchedule = (key: keyof newSchedule, value: string | Schedule[]) => {
-    setNewSchedule((prev) => ({ ...prev, [key]: value }));
+  const updateDateOfScheduleList = (value: Schedule[]) => {
+    setSchedule((prev) => ({ ...prev, dateOfScheduleList: value }));
+  };
+
+  const createScheduleMutation = useMutation({
+    mutationFn: async ({ data, uuid }: { data: newSchedule; uuid: string }) => {
+      if (accessToken) {
+        return mutations.memberSchedule.createMemberSchedule.mutationFn(data, uuid);
+      } else {
+        return mutations.nonMemberSchedule.createNonMemberSchedule.mutationFn(data, uuid);
+      }
+    },
+    onSuccess: ({ scheduleUuid }) => {
+      navigate(`/${uuid}/share`, { state: { scheduleUuid } });
+    },
+  });
+
+  const handleSubmitMeeting = async () => {
+    if (!uuid) return;
+
+    const data = accessToken ? schedule : { ...schedule, nickname: nickName };
+    createScheduleMutation.mutate({ data, uuid });
   };
 
   return (
     <Funnel>
       <Funnel.Step name="닉네임설정">
-        <NickNameForm
-          // TODO : 일정 목록 페이지로 Navigate
-          onPrev={() => {}}
-          onNext={() => setStep('일정입력')}
-          value={newSchedule.nickName}
-          setValue={(value: string) => updateSchedule('nickName', value)}
-        />
+        {!accessToken && (
+          <NickNameForm
+            onPrev={() => navigate(`/${uuid}`)}
+            onNext={() => setStep('일정입력')}
+            value={nickName}
+            setValue={(value: string) => setNickName(value)}
+          />
+        )}
       </Funnel.Step>
 
       <Funnel.Step name="일정입력">
-        <ScheduleInputForm
-          onPrev={() => setStep('닉네임설정')}
-          onNext={() => {}}
-          setValue={(value: Schedule[]) => updateSchedule('dateOfScheduleList', value)}
-        />
+        {step === '일정입력' && (
+          <ScheduleInputForm
+            uuid={uuid as string}
+            onPrev={() => {
+              if (!accessToken) {
+                setStep('닉네임설정');
+              } else if (accessToken) {
+                navigate(`/${uuid}`);
+              }
+            }}
```

Comment:
삼항으로 줄여도 괜찮을 것 같다는 생각이 듭니당 아니면 `if(accessToken)` 만 작성하는게 더 잘 읽힐 것 같아유

```suggestion
            onPrev={() => accessToken ? navigate(`/${uuid}`} : setStep('닉네임설정')
```